src/ScopeRequest.js

```diff
@@ -311,7 +317,18 @@ class ScopeRequest {
     return true;
   }
 
-  constructor(uniqueId, requestedItems, channelsConfig, appConfig, partnerConfig, authentication = true, mode = 'ADVANCED') {
+  static async create(uniqueId, requestedItems, channelsConfig, appConfig, partnerConfig, authentication = true, mode = 'ADVANCED') {
```

Comment:
Static async method... some functions in the ScopeRequest constructor are now async, so moved them in here..

Reply:
This is the new way to create a scope request